portfolio/src/main/webapp/script.js

```diff
@@ -67,3 +63,11 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/** Tells the server to delete the comments. */
+function deleteComments() {
+  const params = new URLSearchParams();
+  fetch('/delete-task', {method: 'POST', body: params});
+  var myobj = document.getElementById('comment-container');
+  myobj.remove();
```

Comment:
optional: calling fetchComments() would make this code simpler, although it would be less efficient